app/src/main/java/com/keylesspalace/tusky/components/compose/ComposeViewModel.kt

```diff
@@ -71,13 +71,13 @@ class ComposeViewModel @Inject constructor(
     private var scheduledTootId: String? = null
     private var startingContentWarning: String = ""
     private var inReplyToId: String? = null
-    private var originalStatusId: String? = null
     private var startingVisibility: Status.Visibility = Status.Visibility.UNKNOWN
 
     private var contentWarningStateChanged: Boolean = false
     private var modifiedInitialState: Boolean = false
     private var hasScheduledTimeChanged: Boolean = false
 
+    var originalStatusId: String? = null
```

Comment:
This exposes the variable now?

Reply:
Yes, so the activity can put it into the result

Reply:
I see. That is done for other variables here as well...